metrics-operator/converter/slo_converter.md

```diff
@@ -0,0 +1,303 @@
+# SLO Converter
+
+## Description
+
+SLO converter is a tool to convert `slo.yaml` files used in [KeptnV1](https://v1.keptn.sh/) into the new
+`AnalysisDefinition` resources used in the new kubernetes-native [Keptn](https://lifecycle.keptn.sh/).
+The converter is part of `metrics-operator` image.
+
+## Usage
+
+The converter will convert a single `slo.yaml` file into single `AnalysisDefintion` resource.
+
+To run the converter, execute the following command:
+
+```shell
+docker-run <METRICS_OPERATOR_IMAGE> manager --convert-slo=<PATH_TO_SLO> --slo-namespace=<ANALYSIS_VALUE_TEMPLATE_NAMESPACE> --definition=<DEFINITION_NAME>
+```
+
+Please be aware, you need to substitute the placeholders with the following information:
+
+* **PATH_TO_SLO** - path to your `slo.yaml` file
+* **ANALYSIS_VALUE_TEMPLATE_NAMESPACE** - namespace of `AnalysisValueTemplate` which will be referenced in objectives
+* **DEFINITION_NAME** - name of created `AnalysisDefinition`
+
+> **Note**
+
+All the SLOs present in `slo.yaml` file will reference `AnalysisValueTemplate` resources from the namespace defined
+by `ANALYSIS_VALUE_TEMPLATE_NAMESPACE` argument.
+
+## Conversion details
+
+We have multiple use-cases which are and which are not supported.
+There is a need to convert the use-cases that make
+logical sense and are common, but in some cases, where it is problematic and these cases will not be supported.
+
+> **Note** Please be aware, that comparison criteria containing `%` symbol ware not supported and will be ignored.
+
+### Unsupported use-cases
+
+Criteria with 3 and more inputs won't be supported, only the first 2 non-percentage inputs (those not containing `%`
+as we do not support comparison rules) will be taken and converted.
+In the example below, only `<600` and `>400` rules will be converted.
+Rule `>800` will be ignored.
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria
+    - "<600"
+    - ">400"
+    - ">800"
+  weight: 2
+  key_sli: true
+```
+
+Secondly, conversion of multiple criteria elements won't be supported at all.
+These criteria are combined with logical OR operator (see documentation [here](https://github.com/keptn/spec/blob/master/service_level_objective.md#objectives)).
+Only a single criteria element rules will be supported.
+These criteria will be converted into informative objectives.
```

Comment:
This might be a bit confusing - does "these" refer to the single criteria element rules mentioned in the sentence before that, or to the multiple criteria elements? I guess the latter, but we should be clearer here. Maybe something like:

Second, only rules containing a single criteria element will be supported. Rules containing multiple criteria elements can not be converted and will be turned into informative items.
Informative in this context means that there are no objectives that have to be met and therefore the item will not affect the overall score. However, their values are still retrieved to provide additional insights for the execution of an Analysis.